PROJECT

```diff
@@ -0,0 +1,66 @@
+domain: security-profiles-operator.x-k8s.io
+layout:
+- go.kubebuilder.io/v3
+plugins:
+  manifests.sdk.operatorframework.io/v2: {}
+  scorecard.sdk.operatorframework.io/v2: {}
+projectName: security-profiles-operator
```

Comment:
Is this the name displayed to the user? Then I'd suggest 
```suggestion
projectName: Security Profiles Operator
```

Reply:
This is the name of the CSV and must follow the usual "subdomain rules":
```
ERRO[0000] ClusterServiceVersion validation: [CSVFileNotValid] (Security-profiles-operator.v0.4.2-dev) metadata.name "Security-profiles-operator.v0.4.2-dev" is invalid: a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*') 
FATA[0000] Error generating bundle manifests: error generating ClusterServiceVersion: invalid generated ClusterServiceVersion 

```